README.md

```diff
@@ -32,8 +32,7 @@ $bool = $validator->isValid('NL123456789B01');
 ```
 
 Additionally check whether the VAT number is in use, with a call to the [VAT
-Information Exchange System (VIES)]
-(http://ec.europa.eu/taxation_customs/vies/faq.html#item_16) SOAP web service:
+Information Exchange System (VIES)](http://ec.europa.eu/taxation_customs/vies/faq.html#item_16) SOAP web service:
```

Comment:
They are identical. 😁 Did you mean `https://...` perhaps?

Reply:
Yeah, it was the markdown for the link which was broken (not the actual URL) - the link was just automatically linked on the URL by github.

Reply:
Ah, good catch!